packages/minify-html/README.md

```
@@ -0,0 +1,75 @@
+  output: 'bar.html',
+  options: {
+    keep_comments: true,
+    keep_closing_tags: true,
+    preserve_brace_template_syntax: true
+  }
+});
+```
+
+## Documentation
+
+Visit https://node-minify.2clics.net for full documentation.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)